src/world.rs

```diff
@@ -774,6 +775,41 @@ impl World {
         self.archetypes_inner().iter()
     }
 
+    /// Faster way of accessing a Component many times
+    pub fn column<T: Component>(&self) -> Column<'_, T> {
+        let archetypes = self.archetypes.archetypes.as_slice();
+        let entities = self.entities.meta.as_slice();
+        let mut archetype_column_offsets: Vec<Option<usize>> = Vec::new();
+        for i in archetypes.iter() {
+            archetype_column_offsets.push(i.get_state::<T>());
+        }
```

Comment:
Could this work be moved inside `Column::new`? That would help limit the growth of this already large module, and save an argument.